cc/postgres/docker/README.md

```diff
@@ -0,0 +1,41 @@
+This image is based upon [`postgres:12`](https://hub.docker.com/_/postgres/) and adds PostgreSQL extension with several epsilon-DP aggregate functions. 
```

Comment:
It would also be good to add a note about the existence of the docker image to cc/postgres/README.md

Reply:
​workspace​(​name​ ​=​ ​"cpp_exp"​) 
 ​load​( 
 ​    ​"@bazel_tools//tools/build_defs/repo:git.bzl"​, 
 ​    ​"git_repository"​, 
 ​) 
  
 ​git_repository​( 
 ​    ​name​ ​=​ ​"com_google_absl"​, 
 ​    ​remote​ ​=​ ​"https://github.com/abseil/abseil-cpp.git"​, 
 ​    ​tag​ ​=​ ​"20200225.2"​, 
 ​) 
  
 ​git_repository​( 
 ​    ​name​ ​=​ ​"com_google_googletest"​, 
 ​    ​remote​ ​=​ ​"https://github.com/google/googletest.git"​, 
 ​    ​tag​ ​=​ ​"release-1.8.1"​, 
 ​)